### ISSUE TITLE
Can't launch Cloud Compare on Manjaro Linux

### ISSUE THREAD

**Issue by segdan** [CloudCompare/CloudCompare#630]
## Current Behavior
I've just installed cloudcompare through snap, but when I try to lauch it with cloudcompare.CloudCompare I get this error:

```
Gtk-Message: Failed to load module "canberra-gtk-module"
[1]    11585 segmentation fault (core dumped)  cloudcompare.CloudCompare
```

## Context/Comments
libcanberra is installed in my system (both 64 and 32 bit versions).

## Your Environment

* OS & Version: Manjaro kde
* CC Version: 2.10-beta

**Comment by dgirardeau** [comment]
Looks like it's related to http://www.danielgm.net/cc/forum/viewtopic.php?t=2333. Not sure it will help though...

**Comment by segdan** [comment]
yeah I've found that discussion, but sadly it doesn't help very much...I had to compile it from source and now everything seems to be ok. I wonder if the snap package is broken or if is it just my machine that has this problem. Anyway, thank you.

**Comment by rjanvier** [comment]
I need to investigate on this since it's the second report on this... but, as I love dog fooding, I use it on my own computer and it seems to work perfectly until now ;). 
Maybe, first, be sure to have the last version of snapd etc installer on your computer.

**Comment by segdan** [comment]
yes I think I have the latest version of snapd (2.27.6), cause I've installed yesterday only for cloudcompare

**Comment by Nikos-Prinios** [comment]
I have the same issue (archlinux-snap-manjaro). Did you solve yours?

**Comment by segdan** [comment]
Nope, I had to compile it from source to be able to use it

**Comment by knarrff** [comment]
Just one more point of information: It seems to work here: Debian 9, CC snap, 2.9.1 64 bit.

**Comment by rjanvier** [comment]
Hi, beta version is available on Flathub
Please read [this](https://discourse.flathub.org/t/how-to-use-flathub-beta/2111) if you want to know the basic about beta channel on flathub. For CC, basically you have to do:

```
flatpak remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak install flathub-beta org.cloudcompare.CloudCompare
```

You may need to make the beta version the default one with

```
flatpak make-current org.cloudcompare.CloudCompare beta
```
